addons/project/data/project_mail_template_data.xml

```diff
@@ -56,13 +56,13 @@
                     <table style="width:100%;text-align:center;">
                         <tr>
                             <td>
-                                <a href="/rating/${access_token}/10">
```

Comment:
For all the templates that you modified that are in 'noupdate' state, like this one, you should write a migration script to force the update. ;)

Reply:
Done